docs/ServerSide.md

```diff
@@ -79,25 +79,25 @@ Root folders can be re-sequenced using drag/drop in the Explorer view, or by edi
 
 ## Web Application (CSP) Files
 
-To edit web application files (also known as CSP files) on a server, configure the uri as `isfs://myserver{csp_application}?ns=XXX&csp`
+To edit web application files (also known as CSP files) on a server, configure the uri as `isfs://myserver:xxx{csp_application}?csp`
 
-For example, the following uri gives you access to the server-side files of the `/csp/user` application. The `csp`  query parameter is mandatory and the `ns` parameter must specify the correct namespace for the web application.
+For example, the following uri gives you access to the server-side files of the `/csp/user` application. The `csp`  query parameter is mandatory and the namespace suffix on the server name must specify the correct namespace for the web application.
```

Comment:
I think this reads better as "the suffix on the server name must specify the correct namespace". Reading "the namespace suffix on the server name must specify the correct namespace" got me confused a little by the redundancy.